charmcraft/commands/build.py

```diff
@@ -265,17 +261,25 @@ def pack_charm_in_instance(self, instance: Executor, bases_index: int) -> str:
         elif message_handler.mode == message_handler.QUIET:
             cmd.append("--quiet")
 
-        try:
-            instance.execute_run(
-                cmd,
-                check=True,
-                cwd=get_managed_environment_project_path().as_posix(),
-            )
-        except subprocess.CalledProcessError as error:
-            capture_logs_from_instance(instance)
-            raise CommandError(
-                f"Failed to build charm for bases index '{bases_index}'."
-            ) from error
+        logger.info(f"Packing charm {charm_name!r} which may take several minutes...")
```

Comment:
Mentioning minutes gives me a Windows 95 progress bar vibe, won't age well
```suggestion
        logger.info(f"Packing charm {charm_name!r}...")
```

Reply:
Haha, well I expect it to be redone in a month or so with @facundos new messages / status handling.  Will update.

Reply:
Done